Mage.Sets/src/mage/cards/s/SubversiveAcolyte.java

```diff
@@ -0,0 +1,132 @@
+package mage.cards.s;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.Mode;
+import mage.abilities.TriggeredAbilityImpl;
+import mage.abilities.common.ActivateOncePerGameActivatedAbility;
+import mage.abilities.costs.common.PayLifeCost;
+import mage.abilities.costs.mana.GenericManaCost;
+import mage.abilities.dynamicvalue.common.StaticValue;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.SacrificeEffect;
+import mage.abilities.effects.common.continuous.*;
+import mage.abilities.keyword.LifelinkAbility;
+import mage.abilities.keyword.TrampleAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.FilterPermanent;
+import mage.game.Game;
+import mage.game.events.GameEvent;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.targetpointer.FixedTarget;
+
+import java.util.UUID;
+
+/**
+ * @author Svyatoslav28
+ */
+public final class SubversiveAcolyte extends CardImpl {
+
+    public SubversiveAcolyte(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.CREATURE}, "{1}{B}");
+
+        this.subtype.add(SubType.HUMAN);
+        this.power = new MageInt(2);
+        this.toughness = new MageInt(3);
+
+        // {2}: Pay 2 life: Choose one. Activate only once.
+        // *  Subversive Acolyte becomes a Human Cleric. It gets +1/+1 and gains lifelink.
+        Ability ability = new ActivateOncePerGameActivatedAbility(Zone.BATTLEFIELD, new AddCardSubTypeSourceEffect(Duration.Custom, SubType.HUMAN, SubType.CLERIC).setText("{this} becomes a Human Cleric"),
+                new GenericManaCost(2), TimingRule.INSTANT);
+        ability.addEffect(new BoostSourceEffect(1, 1, Duration.WhileOnBattlefield));
+        ability.addEffect(new GainAbilitySourceEffect(LifelinkAbility.getInstance(), Duration.WhileOnBattlefield));
```

Comment:
hmm, why is one of these effects Duration.Custom and the other two Duration.WhileOnBattlefield? I can't remember off the top of my head if both work, but can you make them the same and confirm functionality in game?

also you need to set the text for the second two effects

Reply:
![image](https://github.com/user-attachments/assets/33a8b906-fe10-49d5-9c86-9bc6b8aebfef)
Is there any particular reason why I need to use text here? The effect displays correctly on all instances and additional text may even break the current description. Am I missing something?

Reply:
The correct text for the first mode is "Subversive Acolyte becomes a Human Cleric. It gets +1/+1 and gains lifelink."

So the boost effect needs to be "It gets +1/+1" and the gain ability effect "and gains lifelink".